loaderviewlibrary/src/main/java/com/elyeproj/loaderviewlibrary/LoaderTextView.java

```diff
@@ -104,4 +108,64 @@ protected void onDetachedFromWindow() {
         super.onDetachedFromWindow();
         loaderController.removeAnimatorUpdateListener();
     }
+
+    private void checkPlaceholderAttributes(AttributeSet attrs) {
+        TypedArray typedArray = getContext().obtainStyledAttributes(attrs, R.styleable.LoaderTextView, 0, 0);
+        int placeholderTextRes =
+                typedArray.getResourceId(R.styleable.LoaderTextView_placeholder_resource, NO_PLACEHOLDER);
+        String stringArgument =
+                typedArray.getNonResourceString(R.styleable.LoaderTextView_placeholder_string_argument);
+        int intArgument =
+                typedArray.getInt(R.styleable.LoaderTextView_placeholder_int_argument, NO_PLACEHOLDER);
+        float floatPlaceholder = (float) NO_PLACEHOLDER;
+        float floatArgument =
+                typedArray.getFloat(R.styleable.LoaderTextView_placeholder_float_argument, floatPlaceholder);
+
+        if (placeholderTextRes != NO_PLACEHOLDER) {
+            Object argument = null;
+            if (stringArgument != null) {
+                argument = stringArgument;
+            }
+            else if (intArgument != NO_PLACEHOLDER) {
+                argument = intArgument;
+            }
+            else if (floatArgument != floatPlaceholder) {
+                argument = floatArgument;
+            }
+
+            placeholderText = getResources().getString(placeholderTextRes, argument);
```

Comment:
Looks like this is very restricted to a single argument. If one would need to have more arguments, that would not be possible anymore. Having a partial feature might confuse user of the usage.

Reply:
Yes, it is restricted. Afaik it's not possible to pass an array of arguments directly like `app: placeholder_int_arguments="1,2,3"`. The only option would be to pass a reference to an int-array. I didn't like that and therefore added to the readme

> If you need more than one argument you have to use `setPlaceholder(String)` programmatically

However, if this is fine for you I have no problem doing that.